go-fuzz/mutator.go

```diff
@@ -355,8 +362,30 @@ func (m *Mutator) mutate(data []byte, ro *ROData) []byte {
 				lit = []byte(ro.strLits[m.rand(len(ro.strLits))])
 			} else {
 				lit = ro.intLits[m.rand(len(ro.intLits))]
-				if m.rand(3) == 0 {
-					lit = reverse(lit)
+
+				if (len(lit) == 8 || len(lit) == 4 || len(lit) == 2) && m.rand(6) == 0 {
```

Comment:
`m.rand(6)` is the likelihood of triggering this path, right?

Is there any reason why we have 1/6 here and not any other %?

Reply:
Exactly it is the likelihood for triggering it. I am not sure how the other probabilities were chosen so I took something that was a bit less likely than the other ones. However I am not sure with this. Maybe we should add an option to enable the encoding or not. So that if the protocol/program uses LEB128 then it should be encoded and otherwise not. What do you think?

Reply:
I think we can leave it for now